templates/hooks/cluster/sdk_read_many_post_set_output.go.tpl

```diff
@@ -0,0 +1,26 @@
+	if resp.Clusters[0].NumberOfShards != nil {
```

Comment:
nit: early exit for if `len(resp.Clusters) < 1`, then you can just set `cluster := resp.Clusters[0]`